Ollivanders/src/net/pottercraft/Ollivanders2/StationarySpell/ExtraDimensional.java

```diff
@@ -66,11 +64,11 @@ private void createSpace ()
       loc.getBlock().setType(Material.BEDROCK);
    }
 
-   public Set<Block> getBlocksInCube (Location loc)
+   private Set<Block> getBlocksInCube (Location loc)
    {
       Block center = loc.getBlock();
-      int blockRadius = (int) (dimenRadius + 1);
-      Set<Block> blockList = new HashSet<Block>();
+      int blockRadius = dimenRadius + 1;
+      Set<Block> blockList = new HashSet<>();
```

Comment:
nitpick: blockList is a Set

Reply:
Yea, I am not sure if this code works or not so I didn't want to refactor it too far until I know how it works.